RunningButton/RunningButton/RunningButton.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="RunningButton.cs" company="IlyaSotnikov">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace RunningButton
+{
+    using System;
+    using System.Drawing;
+    using System.Windows.Forms;
+
+    /// <summary>
+    /// Running button class.
+    /// </summary>
+    public partial class RunningButton : Form
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RunningButton"/> class.
+        /// </summary>
+        public RunningButton()
+        {
+            this.InitializeComponent();
+        }
+
+        private void RunningButton_Load(object sender, EventArgs e)
+        {
+            this.button.Location = new Point(this.Width / 2, this.Height / 2);
+        }
+
+        private void Button_Click(object sender, EventArgs e)
+        {
+            Application.Exit();
+        }
+
+        private void Button_MouseEnterOrOnButton(object sender, EventArgs e)
+        {
+            Button button = (Button)sender;
+
+            Random random = new Random();
```

Comment:
```suggestion
            var random = new Random();
```
Но вообще, Random не надо создавать каждый раз заново на КАЖДОЕ перемещение мыши над кнопкой. Сборщик мусора спасибо не скажет. Random вообще лучше всего иметь один на целое приложение.